app/hotels/src/allHotels/searchForm/locationPicker/__tests__/__snapshots__/RecentSearches.test.js.snap

```diff
@@ -0,0 +1,46 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`RecentSearches renders 1`] = `
+<UNDEFINED>
```

Comment:
Looks like `react-test-renderer` does not know about `React.Fragment` yet

Reply:
I am fine to accept this. Thanks for pointing it out.

Reply:
This will be fixed in the future version of Jest: https://github.com/facebook/jest/issues/5786